crates/sui-core/src/unit_tests/authority_tests.rs

```diff
@@ -1639,7 +1640,7 @@ async fn test_objected_owned_gas() {
 
 /// Create a `CompiledModule` that depends on `m`
 fn make_dependent_module(m: &CompiledModule) -> CompiledModule {
-    let mut dependent_module = file_format::empty_module();
+    let mut dependent_module = empty_module();
```

Comment:
Nit: I would leave these as `compiled_module::empty_module()` to make the code clearer